### ISSUE TITLE
Images sometimes don't load on 1.5.4

### ISSUE THREAD

**Issue by ISB7** [dinbror/blazy#85]
Hello Support,

I use your beautiful script (js minify 1.5.4) and JS :

<script>var bLazy = new Blazy({selector:'img',container:'#frame1'});bLazy.revalidate();bLazy.destroy();</script>

But sometimes if you refresh the page http://www.insidebasket.com, images don't load... If you refresh again, it's ok. I don't know how to solve this.

Could you help me ?

Thanks you

**Comment by ISB7** [comment]
I think with <script>var bLazy = new Blazy({selector:'img',container:'#frame1'});setTimeout(function(){bLazy.revalidate();},300);</script> it's better...

**Comment by tuurbo** [comment]
I'm getting the exact same issue, reloading page fixes it.

**Comment by dinbror** [comment]
Hey @tuurbo

Do you have a live example, jsfiddle or similar?

**Comment by tuurbo** [comment]
Looks like the image is being downloaded, because i can see it in the chrome tools network tab, but it isn't replacing the src attribute with the image url.

Theres a link inside to a live site with the issue http://pastebin.com/T9p6x21i

**Comment by tuurbo** [comment]
Did some tests on my local env and it seems the success callback isn't being fired for ever image, but the image is being downloaded.

Example:
5 images downloaded successfully, but only got 2 success callbacks.

```
new Blazy({
			selector: '.lazyload-img',
			successClass: 'lazyload-img-loaded',
			success: function(el){
				console.log('success', el);
				$(el).closest('.product-image').removeClass('img-loading');
			},
			error: function(el){
				console.log('error', el);
				$(el).closest('.product-image').removeClass('img-loading');
				$(el).attr('src', '/assets/img/no-product-image-medium.png').addClass('lazyload-img-loaded');
			}
		});
```

**Comment by tuurbo** [comment]
@dinbror I think i found the issue... https://productforums.google.com/forum/#!topic/chrome/p51Lk7vnP2o

Would make sense, since i just started noticing the issue , yet i've been using v1.5.4 for a while..

**Comment by dinbror** [comment]
@tuurbo very interesting, thanks. However I can't reproduce it on your page. Are you on a mac or pc? What OS?

**Comment by tuurbo** [comment]
Windows 10
Chrome 50.0.2661.87

**Comment by ISB7** [comment]
So, same problem with last version of Chrome 50.0.2661.86 (64-bit) on iMac (El Capitan).

Solved with : ;setTimeout(function(){bLazy.revalidate();},300);

**Comment by ISB7** [comment]
finally, not solved :(

**Comment by dinbror** [comment]
Hey @ISB7 and @tuurbo 

Could you please test with version 1.6.1 RC: 
http://dinbror.dk/blazy/versions/blazy-1.6.1.RC.js

**Comment by ISB7** [comment]
Hi dinbror,

I test it : http://www.insidebasket.com/blazy-test.php (you can check this URL)

At the first view, I think it works great. Could you confirm that ?

Is it possible to have a minify JS ?

Thanks you !

**Comment by dinbror** [comment]
Fixed in v.1.6.1

**Comment by dinbror** [comment]
Fixed in chrome v 50.0.2661.94? Will verify later

**Comment by pedro412** [comment]
Hello mate sorry for troubling you but i may have found a bug related to this problem, when first loading the page or visiting for the first time also being cached by browser, it wont load the first images,  reloading the page fixt it i dont know it may be a bug or my particular implementation 
heres the link to the page im working on:

https://camprolabs.com/tutti/

Thanks in advance 👍
